x-pack/solutions/observability/plugins/synthetics/docs/openapi/synthetic_apis.yaml

```diff
@@ -0,0 +1,1446 @@
+                    #     - expression (string): TBD
+                    # status:
+                    #   type: TBD
+                    #   description: A list of expected status codes. 4xx and 5xx codes are considered down by default. Other codes are considered up.     
+            ipv4:
+              type: boolean
+              default: true
+              description: If `true`, ping using the ipv4 protocol.
+            ipv6:
+              type: boolean
+              default: true
+              description: If `true`, ping using the ipv6 protocol.
+            max_redirects:
+              type: number
+              default: 0
+              description: The maximum number of redirects to follow.
+            mode:
+              type: string
+              enum:
+                - all
+                - any
+              default: any
+              description: >
+                The mode of the monitor.
+                If it is `all`, the monitor pings all resolvable IPs for a hostname.
+                If it is `any`, the monitor pings only one IP address for a hostname.
+                If you're using a DNS-load balancer and want to ping every IP address for the specified hostname, you should use `all`.
+            password:
+              type: string
+              description: >
+                The password for authenticating with the server.
+                The credentials are passed with the request.
+            proxy_headers:
+              type: object
+              description: Additional headers to send to proxies during CONNECT requests.
+            proxy_url:
+              type: string
+              description: The URL of the proxy to use for this monitor.
+            response:
+              type: object
+              description: Controls the indexing of the HTTP response body contents to the `http.response.body.contents field`.
+              # properties:
+              #   include_body:
+              #     type: string
+              #     default: on_error
+              #     enum:
+              #       - always
+              #       - never
+              #       - on_error
+              #     description: |
+              #       If it is `on_error`, include the body if an error is encountered during the check.
+              #       If it is `never`, never include the body.
+              #       If it is `always`, always include the body.
+              #   include_body_max_bytes:
+              #     type: number
+              #     default: 1024
+              #     description: Set `response.include_body_max_bytes` to control the maximum size of the stored body contents.       
+            ssl:
+              type: object
+              description: >
+                The TLS/SSL connection settings for use with the HTTPS endpoint.
+                If you don't specify settings, the system defaults are used.
+                    type:
```

Comment:
```suggestion
```

Reply:
Good catch! Fixed in fa994557ff351594bf7cbdccf473bea646391109